docs/neofs-blockstorage.md

```diff
@@ -72,3 +72,44 @@ parameter.
 
 Once all blocks available in the NeoFS container are processed, the service
 shuts down automatically.
+
+### NeoFS Upload Command
+The `upload-bin` command is designed to fetch blocks from the RPC node and upload 
+them to the NeoFS container.
+It also creates and uploads index files. Below is an example usage of the command:
+
+```shell
+./bin/neo-go util upload-bin --cid 9iVfUg8aDHKjPC4LhQXEkVUM4HDkR7UCXYLs8NQwYfSG --wallet-config ./wallet-config.yml --block-attribute "test" --index-attribute "test-index" --rpc-endpoint https://rpc.t5.n3.nspcc.ru:20331 -fsr st1.t5.fs.neo.org:8080 -fsr st2.t5.fs.neo.org:8080 -fsr st3.t5.fs.neo.org:8080
```

Comment:
```suggestion
./bin/neo-go util upload-bin --cid 9iVfUg8aDHKjPC4LhQXEkVUM4HDkR7UCXYLs8NQwYfSG --wallet-config ./wallet-config.yml --block-attribute block --index-attribute index --rpc-endpoint https://rpc.t5.n3.nspcc.ru:20331 -fsr st1.t5.fs.neo.org:8080 -fsr st2.t5.fs.neo.org:8080 -fsr st3.t5.fs.neo.org:8080
```

`--block-attribute` and `--index-attribute` should not be surrounded by quotes. These are string flags.